internal/plugin.go

```diff
@@ -1079,6 +1079,10 @@ func (p *Plugin) assertValidConfig() error {
 					return fmt.Errorf(errorMsgFmt, "severity")
 				}
 
+				if manifest.CustomMessage != nil {
```

Comment:
This is fine if the messages are the same.
```suggestion
				if !reflect.DeepEqual(manifest.CustomMessage, policy.CustomMessage) {
```